src/components/ProteinViewer/index.tsx

```diff
@@ -130,19 +134,19 @@ export const ProteinViewer = ({
   // List of "main" tracks to be displayed, the rest are hidden by default
   const mainTracks = [
     'alphafold confidence',
-    'representative domains',
-    'representative families',
+    'families',
+    'domains',
     'pathogenic and likely pathogenic variants',
     'intrinsically disordered regions',
     'spurious proteins',
-    'residues',
+    'conserved residues',
   ];
 
   const [hideCategory, setHideCategory] = useState<CategoryVisibility>({
     'secondary structure': false,
-    family: false,
-    domain: false,
     'homologous superfamily': false,
```

Comment:
Can be removed as all regions under `homologous superfamily` are now displayed under `domains`.